src/components/ha-yaml-editor.ts

```diff
@@ -88,6 +100,17 @@ export class HaYamlEditor extends LitElement {
         @value-changed=${this._onChange}
         dir="ltr"
       ></ha-code-editor>
+      ${this.copyClipboard
+        ? html`<ha-card outlined>
+            <div class="card-actions">
```

Comment:
You are missing an import for `ha-card` now.

I would just drop `ha-card` and add the 3 lines of css needed:

```css
    border-radius: 0px 0px 12px 12px;
    border: 1px solid var(--divider-color);
    padding: 4px 8px;
```